neuralmonkey/encoders/transformer.py

```diff
@@ -128,9 +139,22 @@ def __init__(self,
                 "If provided, the target space ID should be between 0 and 31. "
                 "Was: {}".format(self.target_space_id))
 
+        if (input_for_cross_attention is None) ^ (n_cross_att_heads is None):
```

Comment:
držel bych se `and` a braní věcí ze `self`, když už tam jsou uložený. Sice bude potřeba to zalomit, ale to se nedá nic dělat.